soc/riscv/riscv-privilege/Kconfig

```diff
@@ -30,4 +30,21 @@ config RISCV_MTVEC_VECTORED_MODE
 	help
 	  Should the SOC use mtvec in vectored mode
 
+config RV_MP_TOTAL_NUM_CPUS
+	int "Total number of CPUs/cores"
+	default 1
+	range 1 5
```

Comment:
Why was this range chosen? Seems artificial

Reply:
CONFIG_MP_NUM_CPUS is limited to 1 to 4 so I set a range of 1 to 5 for this to cover the cases where a monitor core is in the system. There are some RISC-V chips with even more cores but I had to pick something so I at least covered the system I was testing against...